atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/InstrumentedCassandraClient.java

```diff
@@ -68,14 +67,14 @@ public void batch_mutate(String kvsMethodName,
             });
         });
 
-        tablesToCells.forEach((table, numberOfCells) -> getCellsWrittenMeterForTable(table).mark(numberOfCells));
+        tablesToCells.forEach((table, numberOfCells) -> updateCellsWrittenMeterForTable(table, numberOfCells));
     }
 
-    private Meter getCellsWrittenMeterForTable(String table) {
-        return taggedMetricRegistry.meter(
-                MetricName.builder()
+    private void updateCellsWrittenMeterForTable(String table, Long numberOfCells) {
```

Comment:
The method name mentions a Meter which is now slightly misleading.  `updateCellsWrittenForTable ` describes the intent without prescribing a certain implementation.